app/src/main/java/org/wikipedia/analytics/eventplatform/BreadCrumbLogEvent.kt

```diff
@@ -54,7 +56,9 @@ class BreadCrumbLogEvent(
         }
 
         fun logScreenShown(context: Context, fragment: Fragment? = null) {
-            EventPlatformClient.submit(BreadCrumbLogEvent(BreadCrumbViewUtil.getReadableScreenName(context, fragment), "show"))
+            val invokeSource = (fragment?.activity?.intent ?: (context as? Activity)?.intent)?.getSerializableExtra(Constants.INTENT_EXTRA_INVOKE_SOURCE) as? Constants.InvokeSource
+            EventPlatformClient.submit(BreadCrumbLogEvent(BreadCrumbViewUtil.getReadableScreenName(context, fragment),
+                "show" + (invokeSource?.let { ".from." + it.value } ?: "")))
```

Comment:
Can we use `orEmpty()` for the `invokeSource ?: ""`

Reply:
yep!